JenkinsfileImageTest

```diff
@@ -0,0 +1,111 @@
+                ttyEnabled false
+                command ''
+            }
+        }
+    }
+    stages{
+        stage("set build name"){
+            steps{
+                step([$class: 'WsCleanup'])
+                script{
+                    sh("echo \$CI_MESSAGE > ci_message.txt")
+                    env.compose = sh(script: "cat ci_message.txt | jq -r .compose_id", returnStdout: true).trim()
+                    env.release_version = sh(script: "cat ci_message.txt | jq -r .release_version", returnStdout: true).trim()
+                    env.location = sh(script: "cat ci_message.txt | jq -r .location", returnStdout: true).trim()
+                    env.buildname = "${env.release_version} - ${env.compose}"
+                    currentBuild.displayName = "${env.buildname}"
+                }
+            }
+        }
+        imageName = "Fedora-${env.release_version}.qcow2"
+        stage("get image url"){
+            steps{
+                script{
+                    sh """
+                        rm -f images.json 
+                        curl -O ${env.location}/metadata/images.json
+
+                        cat << 'EOF' > path.py
+                        import json
+                        
+                        images = json.load(open('images.json'))['payload']['images']
+                        for image in images:
+                            if image == 'AtomicHost':
+                                continue
+                            if 'x86_64' in images[image]:
+                                for builds in images[image]['x86_64']:
+                                    if builds['format'] == 'qcow2':
+                                        print(builds['path'])
+                        EOF
+                        
+                        python3 ./path.py > path.txt
+                           
+                    """
+
+                    env.img_path = sh(script: "cat ./path.txt", returnStdout: true).trim()
+                    if (env.img_path == "") {
+                        error("Could not find compose")
+                    }
+                    sh("curl -o ${imageName} ${env.location}/${env.img_path}")
+                }
+            }
+        }
+        stage("test compose"){
+            steps{
+                script{
+                    sh("curl -O https://pagure.io/upstream-fedora-ci/raw/master/f/fedora-ci-monitor/validate-test-subject.py")
+                    sh("rm -rf /tmp/artifacts; python3 validate-test-subject.py -s ${imageName}")
+                }
+            }
+        }
+        stage("make sure it runs on test subject"){
+            steps{
+                script{
+                    sh("test /tmp/artifacts && false || true")
+                }
+            }
+        }
+    }
+    post {
+        always {
+            archiveArtifacts artifacts: '*.qcow2', fingerprint: true
```

Comment:
Would be nice if we could implement some hack to only save if success, but I don't think required